tox.ini

```diff
@@ -3,8 +3,6 @@
 #
 #  It's meant to show the use of:
 #
-#  - check-manifest
-#     confirm items checked into vcs are in your sdist
```

Comment:
Removing `check-manifest` seems like it's an independent change here.

I have no great love for including it (my original intention for this project was that it be *minimal*) but I think it should be removed in a separate PR, keeping this PR for the move to `pyproject.toml`.

Reply:
I can't recall why I removed this, it must have not been working with `pyproject.toml` at the time (note that the CI run prior to the commit that removed this failed, but the logs are no longer available), but this appears to work with the latest version of `check-mainifest`, so I reverted it in 8945e5e1426c97c64ce1748d9bd69b1e038c7e30.